docs/modules/ROOT/pages/how-to-guides/proc_maintaining_operator_graphs.adoc

```diff
@@ -0,0 +1,222 @@
+  - name: kubernetes
+    kubernetes:
+      inlined: placeholder
+    attributes:
+      deployment/container-port: 50051
+      deployment/cpuRequest: "100m"
+      deployment/memoryRequest: 512Mi
+      deployment/replicas: 1
+      deployment/storageRequest: "0"
+commands:
+  - id: build-image
+    apply:
+      component: image-build"
+----
+<1> Indicates the component type that {ProductName} builds.
+<2> Indicates the Dockerfile as defined earlier.
+<3> Indicates the tag corresponding to the targetted OCP version as defined earlier.
+
+
+== Adding a component
+Users should manage all the FBC components through a single application. This centralized management enables synchronized updates for graphs across all OpenShift versions.
+
+.Prerequisites
+
+* You have access to link:https://console.redhat.com/beta/hac/application-pipeline[App Studio].
+* You have a Git repository with FBC components.
+* You have to link:https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/Import-code/proc_importing_code/#creating-a-red-hat-container-registry-token[create a Red Hat Container Registry token].
+
+.Procedures
+
+. Go to App Studio and select Create application.
+. In the *Git repository URL* field, enter the URL of your FBC-compatible Git repository.
+. Select *Import code*. The system displays the Configure your components for deployment page.
+. Disable the Default build pipeline.
+. In the Secrets section, select *Add secret*.
+. From the *Secret type* dropdown list, select *Image pull secret*.
+. In the *Select or enter name*, enter *registry-redhat-io-docker*. 
+. In the *Value* field, copy paste or upload the downloaded secret.
+. Select *Create*.
+. Select *Create application*. The system displays a Manage build pipelines pop-up and sends a pull request.
+. Select Merge in GitHub.
+. In GitHub, merge the pull request from the “red-hat-trusted-app-pipeline” bot.
+. Let {ProductName} complete PipelineRun for the upgraded build pipeline.
+
+.Verification
+
+. Go to *Activity > Pipeline* runs.
+
+. Select the most recent *PipelineRun*.
+
+. View the +++ <span class="popup"><code>fbc-label-check</code><span class="popuptext">Ensures that FBC image have all necessary labels.</span> </span>, <span class="popup"><code>fbc-validate</code><span class="popuptext">Checks the existence and functionality of binaries, and runs validation on configs within the image.</span> </span>, and the <code>fbc-related-image-check</code> task within the build pipeline. Ensure that the build process is executing these tasks and not indicating a “skip checks” status. If these tasks show a green checkmark, it indicates that the build process has validated the fbc components against an existing link:https://enterprisecontract.dev/docs/ec-policies/release_policy.html#labels_package[Enterprise Contract] policy. However, if they do not show a green checkmark, it indicates some potential issues in FBC components. +++
```

Comment:
I could be wrong, but doesn't the fbc-label-check task have its own logic? and the contract that we check for it come release time ensures that the was task was run during the build?

Reply:
@arewm WDYT?

Reply:
fbc-label-check is now present as part of EC and is no longer a test at build time.